### PR TITLE
DOCS-8880 Cloud Cost Recommendations Update 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Add four more recommendation types with descriptions on the CCM Recommendations doc page.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 1]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+EBS Over-provisioned Throughput
+: An EBS volume with less than 80% provisioned throughput used for reads and writes.
+
+Over-provisioned DynamoDB Capacity
+: A Provisioned DynamoDB table uses less than 80% of its read and write capacity, more than 80% of the time.
+
+### Rate optimization recommendations
+
+Datadog surfaces resources that are charged at on-demand rates or could benefit from rate optimization, which you can consider modifying to to reduce your costs.
```
Comment (alai97) [inline content/en/cloud_cost_management/recommendations/_index.md:138]:
```suggestion
Datadog surfaces resources that are charged at on-demand rates or could benefit from rate optimization, which you can consider modifying to reduce your costs.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 2]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
-Overprovisioned EBS Volume IOPS
+Over-provisioned EBS Volume IOPS
 : EBS Volumes where the amount of IOPS exceeds what is being used.
 
+### Over-provisioned recommendations
+
+Datadog identifies individual resources that are over-provisioned, which you can consider adjusting the size and configuration to reduce your costs and improve the performance of your resources.
+
+RDS Over-provisioned IOPS
+: An RDS instance using less than 80% of the provisioned IOPS used for reads and writes.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:122]:
```suggestion
: An RDS instance using less than 80% of the provisioned IOPS for reads and writes.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 3]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+EBS Over-provisioned IOPS
+: An EBS volume with less than 80% provisioned IOPS used for reads and writes.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:125]:
```suggestion
: An EBS volume using less than 80% of the provisioned IOPS for reads and writes.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 4]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+EBS Over-provisioned Storage
+: An EBS volume with less than 20% storage usage.
+
+EBS Over-provisioned Throughput
+: An EBS volume with less than 80% provisioned throughput used for reads and writes.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:131]:
```suggestion
: An EBS volume using less than 80% of the provisioned throughput for reads and writes.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 5]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+Datadog identifies individual resources that are over-provisioned, which you can consider adjusting the size and configuration to reduce your costs and improve the performance of your resources.
+
+RDS Over-provisioned IOPS
+: An RDS instance using less than 80% of the provisioned IOPS used for reads and writes.
+
+EBS Over-provisioned IOPS
+: An EBS volume with less than 80% provisioned IOPS used for reads and writes.
+
+EBS Over-provisioned Storage
+: An EBS volume with less than 20% storage usage.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:128]:
```suggestion
: An EBS volume with less than 20% of its storage capacity used.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 6]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+EBS Over-provisioned Throughput
+: An EBS volume with less than 80% provisioned throughput used for reads and writes.
+
+Over-provisioned DynamoDB Capacity
+: A Provisioned DynamoDB table uses less than 80% of its read and write capacity, more than 80% of the time.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:134]:
```suggestion
: A provisioned DynamoDB table using less than 80% of its read and write capacity more than 80% of the time.
```

File: content/en/cloud_cost_management/recommendations/_index.md (hunk 1 above)
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:138]:
```suggestion
Datadog identifies resources that are charged at on-demand rates or could benefit from rate optimization. You can consider modifying these resources to reduce your costs. 
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 7]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+### Rate optimization recommendations
+
+Datadog surfaces resources that are charged at on-demand rates or could benefit from rate optimization, which you can consider modifying to to reduce your costs.
+
+Purchase RDS RI
+: An RDS instance older than 45 days old is still charged with on-demand rates.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:141]:
```suggestion
: An RDS instance older than 45 days is still charged with on-demand rates.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 8]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Purchase ElastiCache RI
+: An ElastiCache node older than 45 days old is still charged with on-demand rates.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:144]:
```suggestion
: An ElastiCache node older than 45 days is still charged with on-demand rates.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 9]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Purchase OpenSearch RI
+: An OpenSearch instance older than 45 days old is still charged with on-demand rates.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:147]:
```suggestion
: An OpenSearch instance older than 45 days is still charged with on-demand rates.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 10]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Purchase Redshift RI
+: An Redshift cluster older than 45 days old is still charged with on-demand rates.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:150]:
```suggestion
: An Redshift cluster older than 45 days is still charged with on-demand rates.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 11]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+S3 Intelligent Tiering
+: A bucket’s costs are almost entirely in per-GB standard storage, but GET requests indicate few objects are actually accessed.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:153]:
```suggestion
: A bucket's costs are almost entirely in per-GB standard storage, but GET requests indicate few objects are accessed.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 12]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+S3 Current Version Expiration Lifecycle Rule
+: A standard S3 bucket without a current version expiration lifecycle and does not serve a website contains current version storage bytes older than 30 days old.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:156]:
```suggestion
: A standard S3 bucket without a current version expiration lifecycle and that does not serve a website contains current version storage bytes older than 30 days.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 13]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+S3 Non-current Version Expiration Lifecycle Rule
+: A standard S3 bucket without a non-current version expiration lifecycle and does not serve a website contains non-current version storage bytes older than 30 days old.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:159]:
```suggestion
: A standard S3 bucket without a non-current version expiration lifecycle and that does not serve a website contains non-current version storage bytes older than 30 days.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 14]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Migrate DynamoDB to On-Demand Capacity Mode
+: A provisioned DynamoDB table has an hourly consumption of read and write capacity less than 18% at least once over the last two weeks.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:162]:
```suggestion
: A provisioned DynamoDB table has an hourly read and write capacity consumption below 18% at least once in the last two weeks.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 15]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Migrate DynamoDB to Provisioned Capacity Mode
+: An on-demand DynamoDB table’s hourly consumption of read and write capacity is always greater than 18%.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:165]:
```suggestion
: An on-demand DynamoDB table has an hourly read and write capacity consumption that is always greater than 18%.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 16]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Migrate DynamoDB to Standard Table Class
+: Migrating to the Standard table class offers potential savings from capacity rates compared to the additional costs from storage rates, or it uses the Standard table class' free tier for storage.
+
+Migrate DynamoDB to Infrequent Access Table Class
+: Migrating to the Infrequent Access (IA) table class offers more potential savings from storage rates compared to the additional costs from capacity rates.
+
+### Legacy or unused recommendations
+
+Datadog identifies resources that are running on legacy hardware or are not utilized efficiently, which you can consider upgrading or removing to reduce your costs and improve the performance of your resources.
```
Comment (domalessi) [inline content/en/cloud_cost_management/recommendations/_index.md:175]:
```suggestion
Datadog identifies resources that are running on legacy hardware or are not utilized efficiently. You can consider upgrading or removing these resources to reduce your costs and improve the performance of your resources.
```

**Review by domalessi — APPROVED** [review]
Made some tweaks. Review and use/address as you see fit!

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 17]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+Extended Support for RDS
+: An RDS running an engine version that is no longer supported and incurring [extended support charges][7].
+
+Unused DynamoDB recommendations
+: A DynamoDB table has 0 consumed reads and 0 consumed non-replica writes.
+
+Unused DynamoDB Global Secondary Index
+: A DynamoDB table’s Global Secondary Index (GSI) has 0 consumed reads.
```
Comment (alai97) [inline content/en/cloud_cost_management/recommendations/_index.md:184]:
```suggestion
: A DynamoDB table's Global Secondary Index (GSI) has 0 consumed reads.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 18]
```diff
@@ -105,18 +105,100 @@ Datadog identifies individual resources that are under-utilized or over-provisio
...
+
+### Architecture recommendations
+
+Datadog identifies individual resources related to NAT gateways, which you can consider optimizing to reduce unnecessary charges.
+
+NAT Gateway within VPC transfer charges
+: Resources in the same VPC should avoid communicating with each other through a NAT gateway because that incurs unnecessary NAT gateway processing charges.
+
+NAT Gateway cross-zone transfer charges
+: Resources that need a NAT gateway should use one that is in the same availability zone, or they will incur unnecessary cross-zone transfer charges.
```
Comment (alai97) [inline content/en/cloud_cost_management/recommendations/_index.md:200]:
```suggestion
: Resources that need a NAT gateway should use one that is in the same availability zone, or they can incur unnecessary cross-zone transfer charges.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 19]
```diff
@@ -98,25 +110,87 @@ GP2 EBS Volumes
...
 
-
-Overprovisioned Kubernetes Containers
+Over-provisioned Kubernetes Containers
 : Containers with less than 30% CPU and memory utilization.
 
-
-Overprovisioned EBS Volume Throughput
+Over-provisioned EBS Volume Throughput
 : EBS Volumes where the amount of throughput exceeds what is being used.
```
Comment (alai97) [inline content/en/cloud_cost_management/recommendations/_index.md:127]:
```suggestion
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 20]
```diff
@@ -98,24 +110,83 @@ GP2 EBS Volumes
...
 
-### Rightsize resource recommendations
+Extended Support for RDS
+: An RDS running an engine version that is no longer supported and incurring [extended support charges][7].
+
+### Over-provisioned resource recommendations
 
 Datadog identifies individual resources that are under-utilized or over-provisioned, which you can consider adjusting the size and configuration to reduce your costs and improve the performance of your resources.
 
 Downsize EC2 instances
```
Comment (alai97) [inline content/en/cloud_cost_management/recommendations/_index.md:120]:
```suggestion
Over-provisioned EC2 instances
```

**Pull request merged**